frontend/src/lib/CSVExport.ts

```diff
@@ -2,7 +2,7 @@ import { ColDef } from "ag-grid-community";
 
 export function CSVFormulate(rows: any[], columnDefinitions: ColDef[]) {
   const csvString = [
-    columnDefinitions.map((item) => item.field).join("\t"),
+    columnDefinitions.map((item) => item.headerName).join("\t"),
```

Comment:
This is a cosmetic fix unrelated to MRN search so that exported spreadsheet headers will use the custom `headerName` instead of `field`'s value.